### ISSUE TITLE
Pihole no internet on devices.

### ISSUE THREAD

**Issue by maxlehot1234** [pi-hole/pi-hole#2568]
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*.
- [X] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the the source code relevant to this issue?:**

`2 Not really good`

---
**Expected behaviour:**

`Pi-Hole was supposed to blocks ads and supposed to not block internet`

**Actual behaviour:**

`Internet doesnt work on my iphone when I put the Pi-Hole IP as the DNS. Pihole admin interface see the device , but the device was not able to access the internet`

**Steps to reproduce:**

`Put your pihole IP as DNS for your device. Not sure what is the cause`

**Debug token provided by [uploading `pihole -d` log](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debug):**

`sjs9xtl5gg`

**Troubleshooting undertaken, and/or other relevant information:**

`I try to reboot pihole server, reconfigure everything, reboot my iphone, change Upstream DNS Servers in admin interface settings page.`

Please note that pi-hole was installed on Ubuntu 18.10 and everything was updated (pihole and ubuntu).
I also install and configure pi-vpn correctly and VPN works but again, internet doesn't work on my iphone
I doesn't configfure pihole for my router. I manually apply it to selected device.

Thanks again everyone if you have an idea

**Comment by whvanderpost** [comment]
I'm having the same issue. I use a Airport Extreme as a router and an iMac as a client.
As soon as I enter the IP of the Pi-hole in the DNS field on the Airport Extreme settings I loose the internet connection. When I enter the Google DNS the connection is restored.
After this I tried to manually configure the IP on my iMac and entered the Pi-hole IP in the configuration, also kills the internet connection.
If I ssh into the Pi I can ping the Google DNS IP (which is also configured as the Upstream DNS Server. If I ping a hostname (like www.google.com) from within the SSH session I get no reply.

**Comment by Mcat12** [comment]
@maxlehot1234 I don't see any issues in your log. Are you able to reach http://pi.hole/admin on the device? Do you see queries from the device in the log?

**Comment by maxlehot1234** [comment]
Yeah I see queries for few minutes. After that, connexion "closed", no more internet.

I configure a vpn using pivpn and same problem occured. I see queries trought pihole, but no internet...

**Comment by Mcat12** [comment]
Are you able to reach the web interface via the Pi-hole's IP address?

**Comment by maxlehot1234** [comment]
Yes of course, since I can see all queries made bye the pihole server and my iphones and the vpn, but no internet after like 1-2 minutes on the iphone.

I try for the 5th time to create a new VM for pihole to see if it fix the issue, but I already try 4 VM and same issue. internet doesn't work :(

**Comment by Mcat12** [comment]
When you say no internet, do you mean you can't resolve DNS, or you can't reach internet IP addresses?

**Comment by maxlehot1234** [comment]
I mean, my iphone was not able to reach any internet website. All apps like: facebook, google chrome, twitter, discord, news app, etc are not able to connect to the internet like if the DNS doesn't work...

But the upstream DNS was correctly set to google. I try all upstream DNS from admin panel of pihole and same problem.

I try to restart tmy iphone, restart pihole server, create new pihole vms, reinstall pihole, same issue occured.

If I set the pihole IP adress as the DNS for my computer, I got the same problem. Internet crash and no more website works...

**Comment by Mcat12** [comment]
What is the output of these commands (on a desktop computer using Pi-hole)?
```
nslookup github.com
nslookup pi.hole
```

Make a new debug token after that.

**Comment by Mcat12** [comment]
You said you installed Pi-VPN, but then also followed the VPN guide on the documentation site, which uses OpenVPN instead. What exactly did you do? Your debug log shows that it is set up to listen to DNS requests coming from VPN clients.

**Comment by maxlehot1234** [comment]
I installed pivpn which install openvpn or a fork of it in fact I think.
http://www.pivpn.io/#about


I can switch the VPN to listen the local pihole IP which is xxx.xxx.xxx.15 but I got the same problem if I do this. Noone that use the pihole IP as DNS was able to reach any website.

I change few config and this is the token: p0a5vw3iaf

Even with these things, I am not able to access pihole using: http://pi.hole

But It works with the server IP, so its fine for me, but no idea why internet doesn't work why I connect to the DNS with my iphone for example

**Comment by maxlehot1234** [comment]
Anyone?

I just fresh install pihole without vpn.

And I keep everything by default except the static IP and if I try to use the PIHOLE IP as DNS on my iphone, I have no internet :(

If anyone have an idea, let me know

new token: lrmuyph7ji


Thanks

**Comment by Mcat12** [comment]
I don't see any issues in your debug log, so the issue is most likely in your VPN configuration.

**Comment by maxlehot1234** [comment]
On the latest debug  there is no VPN like I said.
It was just pihole.

Devices still have no internet when connected to the pihole IP as DNS

**Comment by Mcat12** [comment]
The devices are not correctly using Pi-hole. Go on a desktop which is set to use Pi-hole and share the output of this command: `nslookup pi.hole`

**Comment by maxlehot1234** [comment]
The problem is that I try to use it on my iphone and I can't execute commands on the iphone :P 

If I set the DNS on my computer, it works fine, I see few things blocked by the pihole and I cans access the pihole admin using: pi.hole
Output of the nslookup:
C:\Users\MYNAME>nslookup pi.hole
Serveur :   pihole1
Address:  192.168.1.15

Nom :    pi.hole
Address:  192.168.1.15



In my iphone, I use the IP of pihole and boom, no internet. No websites, no news apps, no facebook, no discord, no twitter, etc.

**Comment by Mcat12** [comment]
@e-moe please open a new issue here or on Discourse or Reddit.

@maxlehot1234 do you see DNS requests from the iPhone in the query log?

**Comment by maxlehot1234** [comment]
No, nothing. Like if the iphone was not able to connect

**Comment by Mcat12** [comment]
You might want to contact Apple's support then, since the issue is that the iPhone is not properly using the DNS server you told it to use.

**Comment by maxlehot1234** [comment]
@Mcat12 I'M in contact with apple for this. Hope they will found the issue :P If there is an issue with the iphone. Maybe a problem with manual DNS. I let you know ;) SO please do not close this ticket.

Thanks again for your time

**Comment by maxlehot1234** [comment]
@Mcat12 I just try with another DNS provider (dns adguard DNS IP) and everything works great on my iphone.

So it was an issue with my pihole setup :(

Any idea for this?

**Comment by Mcat12** [comment]
Did Apple shed any light on this? The only difference between adguard and Pi-hole (to the iPhone) is the IP address, so maybe the iPhone doesn't like sending DNS to a local IP address.

**Comment by maxlehot1234** [comment]
@Mcat12 How I can use no-ip to point to that local IP? This also allow me to use pihole on another wifi network

**Comment by Mcat12** [comment]
> How I can use no-ip to point to that local IP?

What do you mean by this?

**Comment by maxlehot1234** [comment]
I mean do you know how I can use this: https://www.noip.com/

I want to use a dynamicDNS that point to my local IP (pihole IP) so I can use the pihole DNS from anywhere.

I can probbaly make it work without any issue, since I use this thing to reach one of my miencraft server for TESTING only. It was host on my computer and have a port that was unlocked on my router.
So using no-ip I am able to make it work and make it accessible from anywhere in the world.



I just want to know if you have an idea or the default port number of pihole, so I can try on my side ;)


Thanks again

**Comment by Mcat12** [comment]
I would not recommend that, as you'll create an [open resolver](http://openresolverproject.org/).

**Comment by maxlehot1234** [comment]
mmm, well I will try few things to make pihole works on my iphone :P 

Apple told me that I can talk to a senior help team, but Is it worth it :P

Since noone seems to have an issue, it was probbaly a bad configuration into pihole :( But which one ahah

**Comment by ManuGithubSteam** [comment]
I have the same issue with a android phone. Apps cant connect to the interet.

**Comment by dschaper** [comment]
@ManuGithubSteam Please open a new issue either here on GitHub or at https://discourse.pi-hole.net

**Comment by M034B5** [comment]
same issue, i can access the pihole (including web) but DNS is not resolving / working.

**Comment by dschaper** [comment]
Please open a new issue either here on GitHub or at https://discourse.pi-hole.net

**Comment by pedro-albuquerque** [comment]
Also same issue - Android device. Debug log snlx3x2sb3 on tricorder

**Comment by dschaper** [comment]
Please use https://discourse.pi-hole.net
